### DISCUSSION TITLE
zh/guide/api/fs

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#304] [Announcements]
# zh/guide/api/fs

POST 新建文件夹 POST /api/fs/mkdir Body 请求参数 请求参数 名称位置类型必选中文名说明 ------------------ Authorizationheaderstring 是 |token Content-Typeheaderstring 否 |none bodybodyobject 否 |none » pathbo...

https://alist.nn.ci/zh/guide/api/fs.html

**Comment by imshuai** [comment]
[POST 获取目录](https://alist.nn.ci/zh/guide/api/fs.html#post-%E8%8E%B7%E5%8F%96%E7%9B%AE%E5%BD%95) 以及 [POST 获取某个文件/目录信息](https://alist.nn.ci/zh/guide/api/fs.html#post-%E8%8E%B7%E5%8F%96%E6%9F%90%E4%B8%AA%E6%96%87%E4%BB%B6-%E7%9B%AE%E5%BD%95%E4%BF%A1%E6%81%AF) 这两个API的请求参数是否书写错误，与List的示例一模一样，但参数说明又不同

**Reply by Kuingsmile** [reply]
示例书写错误，会进行修正

**Comment by joodo** [comment]
`文件类型` 和 `强制刷新` 是什么意思呀？

**Reply by Kuingsmile** [reply]
强制刷新就是强制重新获取文件信息 否则alist会使用缓存的信息

**Comment by falost** [comment]
超过 10m 的文件就会上传失败 不知道有没有遇到过的

**Comment by dr34m-cn** [comment]
https://alist.nn.ci/zh/guide/api/fs.html#post-删除空文件夹

删除空文件夹经测试，在本地存储的引擎下，返回成功，实际没有删除

**Comment by AnsNid** [comment]
POST /api/fs/add_offline_download  可否针对多条 urls 重命名 ; 

url = [http://1.com/1/demo.jpeg, http://1.com/2/demo.jpeg]   这样两条下载回来 有一条被替换掉了.
